### ISSUE TITLE
building from scratch - AesonException, build fails

### ISSUE THREAD

**Issue by holtzermann17** [luna/luna-studio#704]
joe@ubuntu:~$ luna-manager develop luna-studio

Stack is already installed, skipping
Cloning into '/home/joe/luna-develop/apps/luna-studio'...
error running: stack --resolver lts-8.2 --install-ghc runghc --package base --package exceptions --package shelly --package text --package directory --package system-filepath /home/joe/luna-develop/apps/luna-studio/bootstrap.hs
exit status: 1
stderr: Downloading lts-8.2 build plan ...
Downloaded lts-8.2 build plan.
Populating index cache ...
Populated index cache.
mtl-2.2.1: configure
mtl-2.2.1: build
transformers-compat-0.5.1.4: configure
transformers-compat-0.5.1.4: build

[...]

bootstrap.hs: 
Ran commands: 
echo 'installing python locally'
mkdir -p /home/joe/luna-develop/apps/luna-studio/../../tools/python
cd /home/joe/luna-develop/tools/python
git clone https://github.com/pyenv/pyenv.git
which git
prependToPath: /home/joe/luna-develop/apps/luna-studio/../../tools/python/pyenv/bin
prependToPath: /home/joe/luna-develop/apps/luna-studio/../../tools/python/pyenv/shims
pyenv init -
which pyenv
pyenv install 3.6.2
which pyenv
pyenv local 3.6.2
which pyenv
cd /home/joe/luna-develop/apps/luna-studio
pip install --user -r requirements.txt
which pip
cd /home/joe/luna-develop/tools/python
cd /home/joe
echo 'installing node locally'
echo 'NodeTar: node-v6.11.3-linux-x64.tar.gz'
echo 'NodeUrl: https://nodejs.org/dist/v6.11.3/node-v6.11.3-linux-x64.tar.gz'
mkdir -p /home/joe/luna-develop/apps/luna-studio/../../tools/node
cd /home/joe/luna-develop/tools/node
mkdir -p /home/joe/luna-develop/tools/node/6.11.3
wget https://nodejs.org/dist/v6.11.3/node-v6.11.3-linux-x64.tar.gz
which wget
tar -xpzf node-v6.11.3-linux-x64.tar.gz --strip=1 -C 6.11.3
which tar
rm /home/joe/luna-develop/tools/node/node-v6.11.3-linux-x64.tar.gz
cd /home/joe
echo 'installing node modules'
prependToPath: /home/joe/luna-develop/apps/luna-studio/../../tools/node/6.11.3/bin
/home/joe/luna-develop/apps/luna-studio/../../tools/node/6.11.3/bin/npm install less
which /home/joe/luna-develop/apps/luna-studio/../../tools/node/6.11.3/bin/npm
echo 'downloading libraries'
mkdir -p /home/joe/luna-develop/apps/luna-studio/../../
cd /home/joe/luna-develop/
wget https://s3-us-west-2.amazonaws.com/packages-luna/linux/libs/luna-studio-libs.tar.gz
which wget
tar -xpzf ./luna-studio-libs.tar.gz --strip=1
which tar
rm /home/joe/luna-develop/./luna-studio-libs.tar.gz
cd /home/joe
echo 'generate luna shell'
/home/joe/luna-develop/apps/luna-studio/../../tools/stack/stack path --bin-path
which /home/joe/luna-develop/apps/luna-studio/../../tools/stack/stack

Exception: error running: /home/joe/luna-develop/apps/luna-studio/../../tools/stack/stack path --bin-path
exit status: 1
stderr: Downloading lts-10.3 build plan ...
Downloaded lts-10.3 build plan.
AesonException "Error in $.packages.cassava.constraints.flags['bytestring--lt-0_10_4']: Invalid flag name: \"bytestring--lt-0_10_4\""

**Comment by holtzermann17** [comment]
According to Google this is a stack issue that can be fixed by upgrade, but...

```
$ stack upgrade
Current Stack version: 1.6.3, available download version: 1.6.3
Skipping binary upgrade, you are already running the most recent version
```

**Comment by mikusp** [comment]
Thank you for the bug report! We will update `stack` and let you know here.
In the meantime, you can try to workaround the issue by copying your `stack` binary to `/home/joe/luna-develop/tools/stack/` so that luna-manager uses newer version and running `luna-manager develop luna-studio` once again.
